tests/src/Makefile

```diff
@@ -0,0 +1,68 @@
+CFLAGS	?= -O2
```

Comment:
`tests/src/Makefile` repeats a lot of definitions that are in `tests/Makefile`. I don't see why we need to have a separate makefile in `tests/src`. [Recursive make is an antipattern](http://www.real-linux.org.uk/recursivemake.pdf) . Why not put the relevant targets in `tests/Makefile`, the same way `programs/Makefile` has targets for all the program subdirectories?

Reply:
`tests/src/Makefile` has been removed.